examples/mnist_net2net.py

```diff
@@ -218,8 +220,9 @@ def copy_weights(teacher_model, student_model, layer_names):
 
 
 # methods to construct teacher_model and student_models
-def make_teacher_model(train_data, validation_data, epochs=3):
-    '''Train a simple CNN as teacher model.
+def make_teacher_model():
```

Comment:
Shall we keep the data as arguments instead of referring to them globally? Unless there is some performance related reasons?

Reply:
That was also my feeling with this PR: having explicit arguments makes the code more reusable and easier to read. Globals are bad for code reuse and readability.